ajax/libs/valine/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "valine",
+  "filename": "Valine.min.js",
+  "description": "A simple comment system based on Leancloud.",
+  "version": "1.1.8",
+  "keywords": [
+    "Valine",
+    "Simple Comment",
+    "Comment System",
+    "Based on Leancloud",
+    "Minimalist"
+  ],
+  "author": "xCss <xioveliu@gmail.com> (https://github.com/xCss)",
+  "license": "GPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/xcss/Valine.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/xcss/Valine.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "*.js"
```

Comment:
Let's use `Valine*.+(js|map|css)`
We use include mode here, and most of time, we's like to get the filename in prefix.
Many thanks.